### DISCUSSION TITLE
is there an api to get contract chain for a commodity futures underlying?

### DISCUSSION THREAD

**Discussion by quant2008** [nautechsystems/nautilus_trader#1074] [General]
is there an api to get contract chain for a commodity futures underlying?

**Comment by cjdsellers** [comment]
This isn't currently built-in, however there has been some related discussion in some other threads. The capability is easily possible.

**Reply by quant2008** [reply]
I searched discussion and issues and could not find the discussion. could you point me to the discussion?
